### PR TITLE
SW-5653 Implemented nav items and stub pages

### PR DESCRIPTION
**Pull request by tommylau523**
Notable TODO work, that will be tracked separately:
1) Green checkmarks for completed sections
2) Deliverable screen matchPath
3) Grey'ed out items when it is not available yet

### CODE REVIEW

**Comment by tommylau523** [comment]
* **#2853** <a href="https://app.graphite.dev/github/pr/terraware/terraware-web/2853?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @tommylau523 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: src/scenes/ApplicationRouter/portal/ApplicationPage.tsx [hunk 1]
```diff
@@ -16,6 +18,17 @@ type Prop = {
 const ApplicationPage = ({ children, title }: Prop) => {
   const { goToHome } = useNavigateTo();
 
+  const { allApplications, setSelectedApplitcation } = useApplicationData();
```
Comment (joshbuchea) [inline src/scenes/ApplicationRouter/portal/ApplicationPage.tsx:21]: There's a spelling typo in `setSelectedApplitcation`

**Review by joshbuchea — APPROVED** [review]
Other than a minor spelling mistake that may have been pre-existing, changes look good 👍

**Comment by tommylau523** [comment]
### Merge activity

* **Jul 15, 11:00 PM EDT**: @tommylau523 merged this pull request with [Graphite](https://app.graphite.dev/github/pr/terraware/terraware-web/2853).

**Pull request merged**